packages/flutter/lib/src/material/app_bar.dart

```diff
@@ -359,6 +368,21 @@ class AppBar extends StatefulWidget implements PreferredSizeWidget {
   /// Defaults to [NavigationToolbar.kMiddleSpacing].
   final double titleSpacing;
 
+  /// The amount of height to be used for the [title].
+  ///
+  /// If not provided, then defaults to [kToolbarHeight].
```

Comment:
Maybe just `The height used for the [title].` and then `This value must not be null, and defaults to [kToolbarHeight].`